docs/kit/logistics/kit-loan-extension.md

```diff
@@ -0,0 +1,27 @@
+# Kit Loan Extension
+
+As per appendix A.2 of the [rules][rules], teams may request to keep the kit after the competition.
+
+## Requesting a loan extension
+
+Instructions for teams to request a loan extension are located in A.2 of the [rules][rules]. Generally this involves emailing Student Robotics before a specific deadline. Requests which come in after this deadline should be denied, unless there are exceptional circumstances.
+
+The rules do not define how long the loan extension can be given for. Historical this has been anywhere to a few weeks after the competition to 2 months. This is the date the kit should be returned, rather than when the teams dispatch it.
+
+If a team is granted a loan extension, make sure they are aware, and explicitly note when the kit is expected back.
+
+[Example email for kit loan extension](https://github.com/srobo/team-emails/blob/master/SR2019/2019-03-11-competition-information.md#kit-loan-extension)
+
+## Kit Return Pack
+
+Teams who have requested loan extensions should be issued with kit return packs at the end of the competition. The pack is mostly comprised of jiffy bags suitable for the fragile boards, and stickers. Teams should be instructed to securely pack and plastic wrap the RUBs for shipping, ensuring the contents can't roll around.
```

Comment:
Could we cross link to the details of this? I believe they're in the runbook already, in the kit logistics stuff (possibly under the topic of general kit shipping rather than kit return specifically). If not, there's either a separate repo for them and/or we've missed merging something. I've definitely seen detailed instructions for this _somewhere_.